lib/locales/fr/frontend/navigation.json

```diff
@@ -48,5 +48,6 @@
   "footer_menu_terms": "Conditions générales",
   "footer_menu_privacy": "Politique de confidentialité",
   "footer_description": "La plateforme pour Lancer et Rejoindre des Hackathons",
-  "copyright": "Tous droits réservés. Propulsé par"
+  "copyright": "Tous droits réservés. Propulsé par",
+  "bug_bounty": "Prime aux Bogues"
```

Comment:
```suggestion
  "bug_bounty": "Programme de Bug Bounty"
```

Reply:
A quick search says that `Prime aux Bogues` is more common. Would you rather continue with your suggestion?

<img width="443" alt="image" src="https://github.com/layerx-labs/taikai-locales/assets/16419361/8eefd71f-2277-4b8c-b175-1a36e9ae7a59">